src/popup/components/AccountSwiper.vue

```diff
@@ -68,9 +70,10 @@ import {
 } from 'vue';
 import { RouteLocation } from 'vue-router';
 import { Swiper, SwiperSlide } from 'swiper/vue';
-import SwiperCore, { Virtual } from 'swiper';
 import { getAddressColor } from '@/utils';
-import { PROTOCOL_AETERNITY } from '@/constants';
+import { IS_MOBILE_APP, PROTOCOL_AETERNITY } from '@/constants';
+import SwiperCore from 'swiper';
+import { Virtual } from 'swiper/modules';
```

Comment:
1. Why this has changed? Previous way of importing looks slimmer?
2. Please move these lines up to group them with external imports.

Reply:
We are using a newer version of Swiper where this has changed